src/server/terrain/biomes.zig

```diff
@@ -38,9 +38,13 @@ pub const SimpleStructureModel = struct { // MARK: SimpleStructureModel
 			std.log.err("Couldn't find structure model with id {s}", .{id});
 			return null;
 		};
+		const vtableModel = vtable.loadModel(parameters) orelse {
+			std.log.err("Internal error occurred attempting to load structure with id {s} Dropping model from biome.", .{id});
```

Comment:
It's not an internal error, instead it's (hopefully) an error that got nicely printed in the lines before.
```suggestion
			std.log.err("Error occurred while loading structure with id '{s}'. Dropping model from biome.", .{id});
```

Reply:
Done